integrations/react/src/components/RecurringCIT.jsx

```diff
@@ -0,0 +1,155 @@
+           family: "AxLLCircular",
+           weight: 400
+         }
+       ]
+       // For more detailed documentation at https://github.com/airwallex/airwallex-payment-demo/tree/master/docs#loadAirwallex
+     }).then(() => {
+       // STEP #4: Create and mount the individual cvc element
+       const cvcEle = createElement("cvc");
+ 
+       // STEP #5: Mount split cvc elements
+       cvcEle.mount("cvc"); // Same as above
+     });
+ 
+     // STEP #7: Add an event handler to ensure the element is mounted
+     const onReady = (event) => {
+       const { type } = event.detail;
+       if (type === "cvc") {
+         setCvcReady(true);
+       }
+     };
+ 
+     // STEP #8: Add an event listener to listen to the changes of the input fields
+     const onChange = (event) => {
+       const { type, complete } = event.detail;
+       if (type === "cvc") {
+         setCvcComplete(complete);
+       }
+     };
+ 
+     window.addEventListener("onReady", onReady);
+     window.addEventListener("onChange", onChange); // Can also use onBlur
+ 
+     return () => {
+       window.removeEventListener("onReady", onReady);
+       window.removeEventListener("onChange", onChange);
+     };
+   }, []); // This effect should ONLY RUN ONCE as we do not want to reload Airwallex and remount the elements
+ 
+   // STEP #6a: Add a button handler to trigger the payment request
+   const handleConfirm = async () => {
+     setIsSubmitting(true); // Example: set loading state
+     setErrorMessage(""); // Example: reset error message
+     const cvc = getElement("cvc");
+     confirmPaymentIntent({
+       element: cvc, // Only need to submit CardNumber element
+       id: intent_id,
+       client_secret,
+       payment_consent_id
+     })
+       // STEP #6b: Listen to the request response
+       .then((response) => {
+         /**
+          * ... Handle event on success
+          */
+         setIsSubmitting(false); // Example: reset loading state
+         window.alert(`Confirm success with ${JSON.stringify(response)}`);
+       })
+       // STEP #6c: Listen to errors
+       .catch((error) => {
+         /**
+          * ... Handle event on error
+          */
+         setIsSubmitting(false); // Example: reset loading state
+         setErrorMessage(error.message ?? JSON.stringify(error)); // Example: set error message
+         console.error("There was an error", error);
+       });
+   };
+ 
+   // Example: Custom styling for the inputs, can be placed in css
+   const inputStyle = {
+     border: "1px solid",
+     borderRadius: "5px",
+     padding: "5px 10px",
+     marginTop: "8px",
+     height: "28px"
+   };
+ 
+   return (
+     <div>
+       <h2>Recurring CIT integration</h2>
+       {/* Example: set loading state before elements are ready */}
+       {!cvcReady && <p>Loading...</p>}
+       {/* Example below: display response message block */}
+       {errorMessage.length > 0 && <p id="error">{errorMessage}</p>}
```

Comment:
```suggestion
       {errorMessage && <p id="error">{errorMessage}</p>}
```

Reply:
not the same, In your suggestion, [] would pass, but here we want to make sure it at least has one error.